games-util/lutris/lutris-9999.ebuild

```diff
@@ -49,14 +52,10 @@ src_install() {
 	distutils-r1_src_install
 }
 
```

Comment:
You can add tests, they work with dev-python/nose
```
python_test() {
    virtualx nosetests -v
}
```

Reply:
Yup, they fire up a xvfb in their travis instance, so I just ran for virtualx.eclass, now that you mention it, I haven't run any test *without* virtualx.